api/advanced-events.yaml

```diff
@@ -0,0 +1,36 @@
+paths:
+  /event/subscribe:
```

Comment:
Can we get a rough idea what this endpoint looks like in the generated protobuf definition?

I'm hoping this+callback maps more stragihtforward to native grpc streaming constructs?

Reply:
The POST /event/subscribe API call will translate into an rpc called subscribe_event (operationId) with the request payload as streaming input and the callback request payload as streaming output. 
```
rpc subscribe_event(stream Event.Subscription) returns (stream Event)
```

Reply:
Sounds good.  Is this reflected in the generated artifact for org.proto?  The last I checked, it's returning details as opposed to stream Event.  Is there change to the openapi->proto generator required?

Reply:
Its not reflected at this time, once agreement is reached a change to the patch process will need to be made.